stable/rabbitmq/README.md

```diff
@@ -90,11 +90,7 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `ldap.server`                                | LDAP server                                      | `""`                                                    |
 | `ldap.port`                                  | LDAP port                                        | `389`                                                   |
 | `ldap.user_dn_pattern`                       | DN used to bind to LDAP                          | `cn=${username},dc=example,dc=org`                      |
-| `ldap.tls.enabled`                           | Enable TLS for LDAP connections                  | `false`                                                 |
-| `ldap.tls.caCertificate`                     | CA certificate for LDAP connections              | `nil`                                                   |
-| `ldap.tls.serverCertificate`                 | Server certificate for LDAP connections          | `nil`                                                   |
-| `ldap.tls.serverKey`                         | Server key for LDAP connections                  | `nil`                                                   |
-| `ldap.tls.existingSecret`                    | Existing secret with certificate content to LDAP credentials   | `nil`                                     |
+| `ldap.tls.enabled`                           | Enable TLS for LDAP connections                  | `false` (if set to true, check advancedConfiguration)   |
```

Comment:
You said "if set to true, check advancedConfiguration"
Where do I check it? **values.yaml** or is this a section of the **README.md**? It's a bit confusing for users